tests/cli/validation/invalid-manifest.stderr

```diff
@@ -1,31 +1,5 @@
 - Load subgraph from subgraph.yaml
 ✖ Failed to load subgraph from subgraph.yaml: Error in subgraph.yaml:
 
-  Path: specVersion
```

Comment:
I like this test, can't we keep both? One for when the protocol is unknown and the other for when the validation of one of them failed.

Reply:
Sure I thought there would be other cases covering those, but I can definitely re-add it to be more like it was before (in addition to this current version).